statsmodels/genmod/generalized_linear_model.py

```diff
@@ -431,8 +433,8 @@ def loglike_mu(self, mu, scale=1.):
         """
         Evaluate the log-likelihood for a generalized linear model.
         """
-        return self.family.loglike(mu, self.endog, self.exog, self.iweights,
-                                   scale)
+        return self.family.loglike(self.endog, mu, self.var_weights,
```

Comment:
Do we want have a default scale that's not 1? Additioanlly, should we use `sum(freq_weights)` as the divisor for scale for the OLS special case (identity-Poisson) and `df_resid` in other cases? 

Unifying this with `llf` in `GLMResults` would be nice.